src/appConfig/appConfig.dev.json

```diff
@@ -1,6 +1,6 @@
 {
     "developmentMode": true,
-    "hostMode": "electron",
+    "hostMode": "browser",
```

Comment:
is it just for testing purpose? We may want to keep the mode 'electron' for convinience to debug the main platform behavior.